### DISCUSSION TITLE
Unable to install Pennylane, eventhough Pennylane has Python  Wheel 3

### DISCUSSION THREAD

**Discussion by mushahidkhan835** [pyscript/pyscript#1896] [Q&A]
Hi i am trying to use Pennylane in my pyscript code but every time i add it  I get the following error (PY1001): Unable to install package(s) 'pennylane'. Reason: Can't find a pure Python 3 Wheel for package(s) 'pennylane'. See: https://pyodide.org/en/stable/usage/faq.html#micropip-can-t-find-a-pure-python-wheel for more information. Pennylane has a Python Wheel. I am using the following to add Pennylane as a Package. What could be causing this?
`    <py-config> 
        packages = [ "pennylane"]
     </py-config>`
I have also tried to uppercase the p in pennylane.

**Comment by WebReflection** [comment]
I'm afraid pennylane is not part of this list: https://pyodide.org/en/stable/usage/packages-in-pyodide.html

This looks like an issue worth filing in *pyodide* project as we don't get to control which wheel lands in there.
